### PR TITLE
added rustlang/atom-ide-rust

### PR DESCRIPTION
**Pull request by OAyomide**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -550,6 +550,7 @@ See also [Are we (I)DE yet?](https://areweideyet.com/) and [Rust Tools](https://
 
   * [Atom](https://atom.io/)
     * [zargony/atom-language-rust](https://github.com/zargony/atom-language-rust)
+    * [rust-lang/atom-ide-rust](https://github.com/rust-lang/atom-ide-rust)
```
Comment (palfrey) [inline README.md:553]: Thanks for the submission. Can you please add a description for the repository (I know the other one above it doesn't have one, but we're requiring this for newer submissions now) and add the Travis build badge for it as I'm seeing a build at https://travis-ci.com/github/rust-lang/atom-ide-rust/builds.

Comment (OAyomide) [reply README.md]: Hi. I have made the changes. Pls take a look

File: README.md [hunk 2]
```diff
@@ -550,6 +550,7 @@ See also [Are we (I)DE yet?](https://areweideyet.com/) and [Rust Tools](https://
 
   * [Atom](https://atom.io/)
     * [zargony/atom-language-rust](https://github.com/zargony/atom-language-rust)
+    * [rust-lang/atom-ide-rust](https://github.com/rust-lang/atom-ide-rust) — Rust IDE support for Atom, powered by the Rust Language Server (RLS) [![Build Status](https://travis-ci.com/rust-lang/atom-ide-rust.svg?branch=master)](https://travis-ci.com/rust-lang/atom-ide-rust)
```
Comment (palfrey) [inline README.md:553]: `https://travis-ci.com/rust-lang/atom-ide-rust.svg?branch=master` 301 redirects to `https://api.travis-ci.com/rust-lang/atom-ide-rust.svg?branch=master` so can you please change over the image url to that please?

Comment (OAyomide) [reply README.md]: Hi. Sorry it took a while! been busy with work.

Made the change. I hope it works now.

**Pull request merged**